src/main/java/uk/gov/digital/ho/hocs/domain/queue/migration/StageTypeMapping.java

```diff
@@ -6,6 +6,7 @@ public class StageTypeMapping {
     final static String MIGRATION_BF2_CASE_CLOSED = "MIGRATION_BF2_CASE_CLOSED";
```

Comment:
> It will be useful to change MIGRATION_BF2_CASE_CLOSED to MIGRATION_BF_CASE_CLOSED before any migration takes place.

Can this be done alongside this change?

Reply:
Done